### PR TITLE
No bug - update the docs about private projects

### PR DESCRIPTION
**Pull request by jotes**
@mathjazz Hey, I'm not sure if that's enough. Should I extend this section and explain the visibility settings in depth?

### CODE REVIEW

File: docs/user/localizing-your-projects.rst [hunk 1]
```diff
@@ -69,9 +69,14 @@ following required fields:
 5. **Download prefix**: used for manually downloading files. Select any
    localizable file on GitHub, click ``Raw`` and replace locale code and the
    following bits in the URL with {locale_code}.
-6. Click **SAVE PROJECT** at the bottom of the page.
-7. After the page reloads, click **SYNC** and wait for Pontoon to import
-   strings. You can monitor the progrss in the Sync log (``/sync/log/``).
+7. Click **SAVE PROJECT** at the bottom of the page.
```
Comment (mathjazz) [inline docs/user/localizing-your-projects.rst:72]: That's actually 6, not 7?

File: docs/user/localizing-your-projects.rst [hunk 2]
```diff
@@ -69,9 +69,14 @@ following required fields:
...
+8. All new projects in Pontoon are private by default and aren't visible to
+   localizers and locale managers.
```
Comment (mathjazz) [inline docs/user/localizing-your-projects.rst:74]: This is not an actual step, so I would instead append it to the last step.

File: docs/user/localizing-your-projects.rst (hunk 1 above)
Comment (adngdb) [reply docs/user/localizing-your-projects.rst:72]: It's doesn't matter much, as when processed, those numbers are not used (they are only cosmetic, for when you read the source file). You could have all items start with `1.`, the end result will be a correctly numbered list.

Comment (mathjazz) [reply docs/user/localizing-your-projects.rst:72]: Yeah, converting all to 1. is even better.

File: docs/user/localizing-your-projects.rst (hunk 2 above)
Comment (jotes) [reply docs/user/localizing-your-projects.rst]: I moved it to the last step, can you check if the phrase still sounds okay?

**Pull request merged**